app/components/SourceSelector.vue.ts

```diff
@@ -5,17 +5,26 @@ import { SourcesService } from 'services/sources';
 import { ScenesService, ISceneItemNode, TSceneNode } from 'services/scenes';
 import { SelectionService } from 'services/selection/selection';
 import { EditMenu } from '../util/menus/EditMenu';
-import SlVueTree, { ISlTreeNode, ISlTreeNodeModel, ICursorPosition } from 'sl-vue-tree';
+import SlVueTree, {
+  ISlTreeNode,
+  ISlTreeNodeModel,
+  ICursorPosition
+} from 'sl-vue-tree';
 
 @Component({
   components: { SlVueTree }
 })
 export default class SourceSelector extends Vue {
-
   @Inject() private scenesService: ScenesService;
   @Inject() private sourcesService: SourcesService;
   @Inject() private selectionService: SelectionService;
 
+  layersTooltip = 'The building blocks of your scene. Also contains widgets.';
```

Comment:
Can we change all these references to "layers" in code to "sources"?  I think an average dev might be very confused about what a layer is if we never make that terminology switch.